packages/assets/express/frontend.js

```diff
@@ -0,0 +1,47 @@
+const fetch = require('isomorphic-unfetch')
+const debug = require('debug')('assets:frontend')
+const { returnImage } = require('../lib')
+const {
+  FRONTEND_BASE_URL
+} = process.env
+
+if (!FRONTEND_BASE_URL) {
+  console.warn('missing env FRONTEND_BASE_URL, the /frontend endpoint will not work')
+}
+
+module.exports = (server) => {
+  server.get('/frontend/:path(*)', async (req, res) => {
+    const {
+      path
+    } = req.params
+
+    if (!FRONTEND_BASE_URL) {
+      console.warn('FRONTEND_BASE_URL not set unable to handle request')
+      return res.status(403).end()
+    }
+
+    const [_, sanitizedPath, webp] = new RegExp(/(.*?)(\.webp)?$/, 'g').exec(path)
+
+    const result = await fetch(`${FRONTEND_BASE_URL}/${sanitizedPath}`, {
+      method: 'GET'
+    })
+      .catch(error => {
+        console.error('frontend fetch failed', { error })
+        return res.status(404).end()
+      })
+    if (!result.ok) {
+      console.error('frontend fetch failed', result.url, result.status)
+      return res.status(result.status).end()
+    }
+
+    return returnImage({
```

Comment:
i'd turn these kind of statements around, it's always easier to follow statements that don't include negations:

if (result.ok) {
return image...
 }
console.error(...
return res.status()

and maybe all the res manipulating stuff could be deduped into little functions like you did with returnImage. returnError(result), returnRedirection(). might help for improved readability